### PR TITLE
[OSCD] Create powershell_cmdline_specific_encoded_methods.yml

### PR DESCRIPTION
**Pull request by vburov**

### CODE REVIEW

File: rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+date: 2020/10/11
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection1:
+        Image|endswith: '\powershell.exe'
+        CommandLine|all:
+            - '*char*'
+            - '*join*'
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml:22]: you need to change `CommandLine|all:` to the `CommandLine|contains|all:` and remove `*` from the `char` and `join` (:

File: rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        Image|endswith: '\powershell.exe'
+        CommandLine|contains:
+            - 'ToChar'
+            - 'ToString'
+            - 'String'
+    selection4:
+        Image|endswith: '\powershell.exe'
+        CommandLine|all:
+            - '*split*'
+            - '*join*'
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml:41]: could you please do the same here?

File: rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    selection5:
+        Image|endswith: '\powershell.exe'
+        CommandLine|all:
+            - '*ForEach*'
+            - '*Xor*'
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml:46]: could you please do the same here?

File: rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml (hunk 1 above)
Comment (vburov) [reply rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml]: Done. At first rule redaction I used this pipe, but during troubleshooting I simplified this construction :-)

File: rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml (hunk 2 above)
Comment (vburov) [reply rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml]: Done.

File: rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml (hunk 3 above)
Comment (vburov) [reply rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml]: Done.

**Comment by yugoslavskiy** [comment]
Thank you very much, @vburov! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**